docs/guide/testing.md

```diff
@@ -1,22 +0,0 @@
-# Testing
```

Comment:
I'm thinking about separating the **usage guides** (for devs who want to import HF into their projects and just use it) from the **contribution guides** (for devs who want to build HF locally and possibly change its source code). I'd like to have a `Contributing` category in the docs (similar to https://reactjs.org/docs/how-to-contribute.html) with pages `Building`, `Testing`, `Contributing` & `Code of Conduct`.

What we want to create in #983 should be a part of **usage guides** so in my opinion it should be separated from the guide about *how to run tests in HF*.

So - for me it doesn't matter that much if the scripts for building and testing are in the same guide or in separate ones but I don't want to mix the content for our users with the content for our contributors.

Reply:
@warpech and one more thing: I changed a few npm scripts. Is it a breaking change? Should it be described in the CHANGELOG?

Reply:
> So - for me it doesn't matter that much if the scripts for building and testing are in the same guide or in separate ones but I don't want to mix the content for our users with the content for our contributors.

Good point, I think this resolves my question.

> @warpech and one more thing: I changed a few npm scripts. Is it a breaking change? Should it be described in the CHANGELOG?

I only consider changed behavior of public JavaScript APIs to be breaking changes. These changes apply to internal project tooling, so I think they are not breaking. 

They can be noted as a change in the changelog, nevertheless.

Reply:
Ok, thanks for your feedback.

> Good point, I think this resolves my question.
Do I get it right that building and testing scripts can stay together in the same docs page?

> Nevertheless, it should be noted as a change in the changelog, in case someone works on a custom build.
Ok, I described the change in the CHANGELOG.md

Reply:
Discussion about the categories of guides moved to #991

Reply:
> Do I get it right that building and testing scripts can stay together in the same docs page?

Yes. 

Generally, I am not a fan of changing the URLs in the docs. I think we should avoid that. But since you have a compelling argument here, I think it is fine